spiped/dispatch.c

```diff
@@ -80,7 +82,7 @@ doaccept(struct accept_state * A)
 	int rc = 0;
 
 	/* If we can, accept a new connection. */
-	if ((A->nconn < A->nconn_max) && (A->accept_cookie == NULL)) {
+	if ((A->nconn < A->nconn_max) && (A->accept_cookie == NULL) && !(A->shutdown)) {
```

Comment:
@cperciva I think you are right :wink: . I just pushed a new commit that calls `network_accept_cancel` and a short tests confirms that spiped indeed does not accept a new connection.